docs/user-guide/global-configurations/filter-condition.md

```diff
@@ -31,8 +31,8 @@ Using filter conditions, you can control the progression of events. Here are a f
     * **Filter Name**: Give a name to the filter.
     * **Description**: (Optional) Add a description to the filter, preferably explaining what it does.
     * **Filter Condition**: You can specify either a pass condition, fail condition, or both the conditions:
-        * **Pass Condition**: Only those events that satisfy the condition will be directed through your pipeline.
-        * **Fail Condition**: Events that fail the condition will not be directed through your pipeline..
+        * **Pass Condition**: Events that satisfy the pass condition will appear in your CD pipeline.
```

Comment:
instead of appear, use something like can be used to trigger or are eligible to trigger

Reply:
Done